tg_bot/__main__.py

```diff
@@ -415,6 +415,34 @@ def migrate_chats(bot: Bot, update: Update):
     LOGGER.info("Successfully migrated!")
     raise DispatcherHandlerStop
 
+def is_chat_allowed(bot, update):
+    if len(WHITELIST_CHATS) =! 0:
+        chat_id = update.effective_message.chat_id
+        if chat_id not in WHITELIST_CHATS:
+            bot.send_message(chat_id=update.message.chat_id,
+                             text='Unallowed chat! Leaving...')
+            bot.leave_chat(chat_id)
+        else:
+            pass
+    elif len(BLACKLIST_CHATS) != 0:
+        chat_id = update.effective_message.chat_id
+        if chat_id in BLACKLIST_CHATS:
+            bot.send_message(chat_id=update.message.chat_id,
+                             text='Unallowed chat! Leaving...')
+            bot.leave_chat(chat_id)
+        else:
+            pass
+    elif len(WHITELIST_CHATS) =! 0 and len(BLACKLIST_CHATS) != 0:
```

Comment:
this if statement is unreachable, since the `len(WHITELIST_CHATS)` will have been caught in the first if statement, and these are in elif blocks.